parkings/tests/api/monitoring/test_region_statistics.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import random
+from collections import OrderedDict
+
+from django.urls import reverse
+from rest_framework import status
+
+from ...utils import create_parkings_and_regions, intersects
+
+list_url = reverse('monitoring:v1:regionstatistics-list')
+
+
+def test_empty(monitoring_api_client):
+    result = monitoring_api_client.get(list_url)
+    assert result.data == {
+        'count': 0,
+        'next': None,
+        'previous': None,
+        'results': [],
+    }
+    assert result.status_code == status.HTTP_200_OK
+
+
+def test_with_single_parking(monitoring_api_client, region, parking):
+    point_in_region = region.geom.centroid
+    parking.location = point_in_region
+    parking.save()
+    assert intersects(point_in_region, region)
+
+    result = monitoring_api_client.get(list_url)
+    assert result.data == {
+        'count': 1,
+        'next': None,
+        'previous': None,
+        'results': [
+            OrderedDict([
+                ('id', str(region.id)),
+                ('parking_count', 1)]),
+        ]
+    }
+    assert result.status_code == status.HTTP_200_OK
+
+
+def test_with_many_parkings_and_specified_time(monitoring_api_client):
+    (parkings, regions) = create_parkings_and_regions(
+        parking_count=20, region_count=5)
+
+    # Pick a point in time that is in between the parking times
+    earliest_start_time = min(x.time_start for x in parkings)
+    earliest_end_time = min(x.time_end for x in parkings)
+    latest_end_time = max(x.time_end for x in parkings)
+    range_start = max(earliest_start_time, earliest_end_time)
```

Comment:
Not sure how earliest start time could ever be after earliest end time :)

Also Faker has `date_time_between()` which could probably help on this(?)

Reply:
Yeah, you're right. earliest_end_time is always the maximum of the two. I'll simplify it. Will also check if date_time_between would be good here.